src/cargo/util/lints.rs

```diff
@@ -1,3 +1,4 @@
+use crate::core::dependency::DepKind;
 use crate::core::FeatureValue::Dep;
```

Comment:
This commit is named after what it does but not the why ("avoid creating implicit features on edition 2024")